src/effects/VST/VSTEffect.cpp

```diff
@@ -603,18 +603,18 @@ BEGIN_EVENT_TABLE(VSTEffect, wxEvtHandler)
 END_EVENT_TABLE()
 
 // Needed to support shell plugins...sucks, but whatcha gonna do???
-intptr_t VSTEffect::mCurrentEffectID;
+intptr_t VSTEffectWrapper::mCurrentEffectID;
 
 typedef AEffect *(*vstPluginMain)(audioMasterCallback audioMaster);
 
-intptr_t VSTEffect::AudioMaster(AEffect * effect,
-                                int32_t opcode,
-                                int32_t index,
-                                intptr_t value,
-                                void * ptr,
-                                float opt)
+intptr_t VSTEffectWrapper::AudioMaster(AEffect * effect,
+                                       int32_t opcode,
+                                       int32_t index,
+                                       intptr_t value,
+                                       void * ptr,
+                                       float opt)
 {
-   VSTEffect *vst = (effect ? (VSTEffect *) effect->ptr2 : NULL);
+   VSTEffectWrapper *vst = (effect ? (VSTEffectWrapper *) effect->ptr2 : NULL);
```

Comment:
Modernize with `static_cast` and `nullptr`

Reply:
ok, done